src/tests/ov_helpers/ov_snippets_models/src/subgraph_customizable.cpp

```diff
@@ -58,6 +58,149 @@ std::shared_ptr<ov::Model> ConvMulActivationFunction::initReference() const {
                                                                   ParameterVector{indata0, indata1}));
     return std::make_shared<ov::Model>(NodeVector{subgraph}, ParameterVector{conv_param, eltwise_param});
 }
+std::shared_ptr<ov::Model> ConvBiasActivationFunction::initOriginal() const {
+    auto conv_param = std::make_shared<op::v0::Parameter>(precision, input_shapes[0]);
+    const auto channels = static_cast<size_t>(input_shapes[0][1].get_length());
+    ov::Shape strides(2, 1);
+    std::vector<ptrdiff_t> pad_begin(2, 1), pad_end(2, 1);
+    const Shape const_shape {channels, channels, 3, 3};
+    const std::vector<float> const_values = ov::test::utils::generate_float_numbers(shape_size(const_shape), -10., 10.);
+    auto weights = std::make_shared<op::v0::Constant>(precision, const_shape, const_values);
+    auto conv = std::make_shared<op::v1::Convolution>(conv_param, weights, strides, pad_begin, pad_end, strides);
+
+    const auto batches = static_cast<size_t>(input_shapes[0][0].get_length());
+    const Shape add_const_shape = {batches, channels, 1, 1};
+    const std::vector<float> add_const_values = ov::test::utils::generate_float_numbers(shape_size(add_const_shape), -10., 10.);
+    auto add_const = std::make_shared<op::v0::Constant>(precision, add_const_shape, add_const_values);
+
+    auto add = std::make_shared<op::v1::Add>(conv->output(0), add_const->output(0));
+    auto unary = custom_ops[1]->clone_with_new_inputs({add->output(0)});
+    add->set_arguments({conv->output(0), add_const->output(0)});
+    unary->set_arguments({add->output(0)});
```

Comment:
When you clone the op with new inputs (on line76-77), you already set new arguments. Thus, L78-79 are extras 😃 
The existing code above with the same extra `set_arguments` was written ~3 years ago: probably, it was needed before (I mot not sure) or we weren't be experienced in ngraph.

Could you please remove extra `set_arguments` in this file?

Reply:
Applied. Thanks Alexandra!